client/components/account-balances/strings.ts

```diff
@@ -21,7 +21,7 @@ export const greetingStrings = {
 
 export const fundLabelStrings = {
 	available: __( 'Available funds', 'woocommerce-payments' ),
-	pending: __( 'Pending funds', 'woocommerce-payments' ),
+	total: __( 'Total balance', 'woocommerce-payments' ),
```

Comment:
Is it serving us to have these strings factored out of the component?

Reply:
There are 3 usages in `client/components/account-balances/index.tsx` so there is benefit of storing it as a 'constant' but not sure having a separate `strings.ts` is a good design.

I see for other component it's structured that way, having a `strings.ts`.

Reply:
I see this pattern is used in our code, but I'm not convinced – to me it makes the code less meaningful to extract the strings (and documentation links, etc). One of the wonderful things about react is you can look at the code/markup and understand what the component looks like and does. Moving strings to constants makes this a little harder, since the reader needs to look up each constant to see what the string is. So when I'm working on components that feel disconnected like this, I consider pulling string back in to make it easier to maintain and more self-contained.

Anyway, I'm just sharing my thoughts – no need to change this if it' working for you :)